tests/testthat/test-lastdose.R

```diff
@@ -192,3 +192,51 @@ test_that("logical comment column is ok", {
   d$C <- sample(c(1, 2), nrow(d), replace = TRUE)
   expect_warning(lastdose(d), msg = "but it wasn't character or logical")
 })
+
+test_that("ii detection issue-21", {
```

Comment:
my preference would probably still be to actually separate this test out at somepoint to be more declarative of what each of these things are targetting. This is where things like the describe(,it()) structure can work out well - but this is not required to get this fix in but would be worth tracking to follow up in some later test refactoring

Reply:
Thanks.  I'll touch base with you on what you'd like to see.